frontend/src/views/Map.vue

```diff
@@ -108,42 +104,63 @@ export default {
   },
 
   methods: {
-
-    setActive(index){
+    setActive(index) {
       this.activeIndex = index;
     },
 
-    setInactive(){
+    setInactive() {
       this.activeIndex = undefined;
     },
 
     navigateToQueuePage(id){
       this.$router.replace("queue/"+id);
     },
-    // receives a place object via the autocomplete component
+
     setPlace(place) {
       this.currentPlace = place;
     },
-    addMarker() {
-      if (this.currentPlace) {
-        const marker = {
-          lat: this.currentPlace.geometry.location.lat(),
-          lng: this.currentPlace.geometry.location.lng()
-        };
-        this.markers.push({ position: marker });
-        this.places.push(this.currentPlace);
-        this.center = marker;
-        this.currentPlace = null;
-      }
-    },
-    geolocate: function() {
-      navigator.geolocation.getCurrentPosition(position => {
+
+    geolocate: function() {  // Gets user's location and centers the map around that 
+      navigator.geolocation.getCurrentPosition((position) => {
         this.center = {
           lat: position.coords.latitude,
-          lng: position.coords.longitude
+          lng: position.coords.longitude,
         };
       });
-    }
-  }
+    },
+
+    search: function() {
+      var queues = []
+
+      if(this.searchItem == null) // Does not make a request if query is empty
+        return;
+
+      axios
+      .get('https://cors-anywhere.herokuapp.com/https://maps.googleapis.com/maps/api/place/nearbysearch/json?location='+this.center.lat+','+this.center.lng+'&radius=1000&name='+this.searchItem+'&key='+process.env.VUE_APP_MAPS_API_KEY)
```

Comment:
Move this call to the separate Api.js, this will enable you to change the base URL for different environments at single place. Also it make sure that the network call configs can be maintained at the single place.